v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/config/defaults/MySqlConfigDefaults.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.source.reader.io.jdbc.iowrapper.config.defaults;
+
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.dialectadapter.DialectAdapter;
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.dialectadapter.mysql.MysqlDialectAdapter;
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.dialectadapter.mysql.MysqlDialectAdapter.MySqlVersion;
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.rowmapper.JdbcValueMappingsProvider;
+import com.google.cloud.teleport.v2.source.reader.io.jdbc.rowmapper.provider.MysqlJdbcValueMappings;
+import com.google.cloud.teleport.v2.source.reader.io.schema.typemapping.UnifiedTypeMapper.MapperType;
+import org.apache.beam.sdk.util.FluentBackoff;
+
+// TODO: Fine-tune the defaults based on benchmarking.
+
+/**
+ * MySql Default Configuration for {@link
+ * com.google.cloud.teleport.v2.source.reader.io.jdbc.iowrapper.JdbcIoWrapper JdbcIoWrapper}.
+ */
+public class MySqlConfigDefaults {
+
+  public static final MapperType DEFAULT_MYSQL_SCHEMA_MAPPER_TYPE = MapperType.MYSQL;
+  public static final DialectAdapter DEFAULT_MYSQL_DIALECT_ADAPTER =
+      new MysqlDialectAdapter(MySqlVersion.DEFAULT);
+  public static final JdbcValueMappingsProvider DEFAULT_MYSQL_VALUE_MAPPING_PROVIDER =
+      new MysqlJdbcValueMappings();
+
+  public static final String DEFAULT_MYSQL_CONNECTION_PROPERTIES =
+      "maxTotal=160;maxpoolsize=160;maxIdle=160;minIdle=160"
+          + ";wait_timeout=57600"
+          + ";interactive_timeout=57600"
+          + ";idletimeout=3600"
```

Comment:
nit: purely speaking due to ocd, the ; looks so much better at the end.

Reply:
I know, but then there's a practice that line breaks should be easy to delete line by line.
For example If I wanted to break `a + b + c + d `
I would probably break it as
```
a
+ b
+ c
+ d
```